examples/locator.js

```diff
@@ -0,0 +1,55 @@
+import launcher from "k6/x/browser";
+
+export default function () {
+  const browser = launcher.launch('chromium', {
+    headless: __ENV.XK6_HEADLESS ? true : false,
+  });
+  const context = browser.newContext();
+  const page = context.newPage();
+  page.goto("https://test.k6.io/flip_coin.php", {
+    waitUntil: "networkidle",
+  });
+
+  /*
+  In this example, we will use two locators, matching a
+  different betting button (head and tail) on the page.
+
+  If you were to query the elements once and save them as
+  below, you would see an error after the initial navigation.
+  Try it!
+
+    const head = page.$("input[value='Bet on heads!']");
+    const tail = page.$("input[value='Bet on tails!']");
+
+  The Locator API allows you to get a fresh element handle each
+  time you use one of the locator methods. And, you can carry a
+  locator across frames. Let's create two locators each locates
```

Comment:
"Across frame navigations"

Reply:
Also, let's make sure the files in the `examples` directory are synced with the ones in the README (minus the `headless` conditional), after addressing any feedback.